website/src/components/Sortable/Sortable.tsx

```diff
@@ -1,7 +1,21 @@
 import { Flex } from "@chakra-ui/react";
-import { closestCenter, DndContext, PointerSensor, TouchSensor, useSensor, useSensors } from "@dnd-kit/core";
+import {
+  closestCenter,
+  DndContext,
+  PointerSensor,
+  TouchSensor,
+  KeyboardSensor,
+  useSensor,
+  useSensors,
+} from "@dnd-kit/core";
 import type { DragEndEvent } from "@dnd-kit/core/dist/types/events";
-import { arrayMove, SortableContext, verticalListSortingStrategy } from "@dnd-kit/sortable";
+import {
+  arrayMove,
+  SortableContext,
+  sortableKeyboardCoordinates,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import { restrictToVerticalAxis } from "@dnd-kit/modifiers";
```

Comment:
small nit, could you move this import up above sortable so it's in order?

Reply:
Sure, just to confirm it should be alphabetically by @scope / unscoped name, then by scoped package name? so:
* a
* @b/a
* @b/e
* c

Reply:
I think with the new eslint change that goes in `@` is always first.  So it'd be

```
@b/a
@b/e
a
c
c-foo
```